src/components/SessionList.vue

```diff
@@ -110,18 +110,18 @@ export default {
 				this.$store.dispatch('setShowAuthorAnnotations', value)
 			},
 		},
-		editorsTooltip() {
-			const tooltipPrefix = t('text', 'Currently active users:') + ' '
-			if (this.sessionPopoverList.length > 0) {
-				const first = this.activeSessions.slice(0, 3).map((session) => session.guestName ? session.guestName : session.displayName).join(', ')
-				const others = this.activeSessions.slice(3).length
-				return tooltipPrefix + first + ' ' + n('text', 'and %n other editor', 'and %n other editors', others)
+		participantsPopover() {
+			if (this.currentSession.guestName) {
+				return this.participantsWithoutCurrent
 			}
-			return tooltipPrefix + this.activeSessions.slice(0, 3).map((session) => session.guestName ? session.guestName : session.displayName).join(', ')
+			return this.participants
 		},
-		activeSessions() {
+		participantsWithoutCurrent() {
+			return this.participants.filter((session) => !session.isCurrent).sort((a, b) => a.lastContact < b.lastContact)
```

Comment:
Why this function is sorting the nbUploadingImages again?

Reply:
Right, pushed an update as it is already sorted in the participants computed prop.